st2mistral/functions/stackstorm.py

```diff
@@ -18,16 +18,104 @@
 from oslo_config import cfg
 from oslo_log import log as logging
 
-from st2mistral import config
-config.register_opts()
-
 from mistral import exceptions as exc
+from st2common.jinja.filters import crypto
```

Comment:
Yep, st2 and mistral have different venvs and mistral currently do not have access to st2common.